drip/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.8.0'
+__version__ = '0.8.1'
```

Comment:
this version should be 0.9.0 because is not a fix, is a non breaking change, right?

Reply:
I believe this might an even minor category because this change does not add new functionality at all.

Reply:
We are using:

```
MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
```

But we can discuss it on the meeting today